service/src/test/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderServiceTest.java

```diff
@@ -201,14 +229,40 @@ void shouldReturnMostRecentUploadedOrderDocumentUrl() {
             returnedMostRecentUploadedOrderDocumentUrl);
     }
 
+    private void assertCommonC21Fields(GeneratedOrder order) {
+        assertThat(order.getType()).isEqualTo(BLANK_ORDER);
+        assertThat(order.getDocument()).isEqualTo(DocumentReference.builder().build());
+        assertThat(order.getOrderDetails()).isEqualTo("Some details");
+        assertThat(order.getOrderDate()).isNotNull();
+        assertThat(order.getJudgeAndLegalAdvisor()).isEqualTo(JudgeAndLegalAdvisor.builder().build());
+    }
+
     @SuppressWarnings("unchecked")
-    private Map<String, Object> expectedData(String date) {
+    private Map<String, Object> expectedData(String date, GeneratedOrderType orderType) {
         ImmutableMap.Builder expectedMap = ImmutableMap.<String, Object>builder();
 
-        expectedMap.put("familyManCaseNumber", "123")
+        switch (orderType) {
```

Comment:
Should we not break this into separate methods?

Reply:
Don't see a need to separate this out as it is quite clean in a switch statement and there are a lot of common fields regardless of chosen order type. We would need a method for each type of order if we split it. Can re-discuss when adding next type of order in later story